lib/main.dart

```diff
@@ -31,6 +31,7 @@ class _RandomWordsState extends State<RandomWords> {
   final _suggestions = <WordPair>[];
   final _saved = <WordPair>{};
   final _biggerFont = const TextStyle(fontSize: 18.0);
+  bool _changeFollowIconColor = false;
```

Comment:
In general, instead of using `bool`, I would use the following:
1) define array with potential colors: `final _colors = [Colors.pink, Colors.orange]`
2) define variable that stores currently used index `var _currentColor = 0`
3) inside build, use color by current index: `color: _colors[_currentColor]`
4) when switching colors, just use cycled increment: `_currentColor = (_currentColor + 1) % _colors.length` to ensure that _currentColor is always less than number of available options

with this approach you can simply add as many colors to your `_colors` as you want, and you don't need to rewrite your code